packages/medusa/src/migrations/1680857773273-drop-variant-id-fk-money-amount.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
question: Should we merge migrations in the PR, or did you intentionally separate them?

Reply:
Yeah, that's what I thought. Fine to keep them as is 👍